code/modules/mob/living/silicon/mommi/examine.dm

```diff
@@ -53,3 +53,38 @@
 		to_chat(fag, "<b>[src] has the following laws:</b>")
 		laws.show_laws(fag)
 		investigation_log(I_GHOST, "|| had its laws checked by [key_name(fag)][fag.locked_to ? ", who was haunting [fag.locked_to]" : ""]")
+
+/mob/living/silicon/robot/mommi/sammi/examine(mob/user)
+
+	var/msg = "<span class='info'>*---------*\nThis is [bicon(src)] \a <EM>[src]</EM>!\n"
+
+	msg += {"<p>This versatile assembler is controlled by a transient mind-cluster network, unfortunately it is prone to connection loss.</p><span class='warning'>"}
+	if (src.getBruteLoss())
+		if (src.getBruteLoss() < 75)
+			msg += "It looks slightly dented.\n"
+		else
+			msg += "<B>It looks severely severely damaged!</B>\n"
+	if (src.getFireLoss())
+		if (src.getFireLoss() < 75)
+			msg += "It looks slightly charred.\n"
+		else
+			msg += "<B>It looks severely burnt and heat-warped!</B>\n"
+	msg += "</span>"
+
+	if(tool_state)
+		var/obj/item/I = tool_state
+		msg += "Its claw is gripping [bicon(I)] [I.gender==PLURAL?"some":"a"] [I.name].\n"
+
+	if(opened)
+		msg += "<span class='warning'>Its cover is open and the power cell is [cell ? "installed" : "missing"].</span>\n"
+	else
+		msg += "Its cover is closed.\n"
+
+	switch(src.stat)
+		if(CONSCIOUS)
+			if(!src.client)	msg += "It appears to be in stand-by mode.\n" //afk
+		if(UNCONSCIOUS)		msg += "<span class='warning'>It doesn't seem to be responding.</span>\n"
+		if(DEAD)			msg += "<span class='deadsay'>It looks completely unsalvageable.</span>\n"
+	msg += "*---------*</span>"
```

Comment:
A lot of copy-paste for a simple examine text increase

Reply:
Removed it all because it was redundant, desc was the same as that line anyways.